inst/rmarkdown/templates/ajs_outbreak/skeleton/skeleton.Rmd

```diff
@@ -583,36 +565,56 @@ linelist_cleaned <- linelist_cleaned %>%
   mutate_at(DATEVARS, linelist::guess_dates,
             error_tolerance = 0.5)
 
-# if you dont have a data dictionary you could do it this way too
+
+
+## Non-DHIS2 data --------------------------------------------------------------
+## Use this section if you did not have DHIS2 data. 
+
+# use the guess_dates() function to make a first pass at date variables.
```

Comment:
if we are going to have commented out code - then commented text should have two ## to be easier to differentiate from code

Reply:
also this way if you just highlight the whole section and ctrl+shift+c then the comments stay comments and code becomes available